libopae++/samples/simple_example.cpp

```diff
@@ -46,25 +46,19 @@ int main(int argc, char* argvp[])
         p.guid = uuid;
     }
     p.bbs_id = 0;
-    auto tokens = token::enumerate({ p });
+    auto tokens = token::enumerate({p});
```

Comment:
Maybe change p to props for clarity?  Whenever I see a variable named p, I default to thinking of it as a pointer variable.